src/liboslexec/llvm_util.cpp

```diff
@@ -474,7 +477,8 @@ static bool initCpuFeatures()
 // So if you change the target cpu or features in liboslexec/CMakeList.txt
 // for any of the wide libraries, please update here to match
 static const char * target_isa_names[] = {
-    "UNKNOWN", "x64", "SSE4.2", "AVX", "AVX2", "AVX2_noFMA", "AVX512", "AVX512_noFMA"
+    "UNKNOWN", "none", "x64", "SSE4.2", "AVX", "AVX2", "AVX2_noFMA",
+    "AVX512", "AVX512_noFMA", "host"
 };
 
 
```

Comment:
We've seen cases where the below requrement of "aes" has prevented the use of the SIMD path because the "aes" cpu feature was turned off in the BIOS.  Since OSL isn't actually using those instructions as far as I know, it should probably be dropped from the requirement list.

Reply:
Thanks for the tip! It sure seems unlikely that those ops would be needed in the JIT.

What would be the reason that somebody would want to turn this off in the BIOS?

Reply:
I think it was a mis-configuration, not intentional.

Reply:
I'll comment out the AES features.

@AlexMWells Can you think of any reason that we would benefit (for OSL JIT purposes) from the AES instructions?

Reply:
Not at this moment, I'm pretty sure one would need to use intrinsics and until we have a use case, we can leave it off.